python/tvm/relay/frontend/pytorch_neo.py

```diff
@@ -0,0 +1,1019 @@
+
+        center = center
+        scale = scale
+
+        return _op.nn.batch_norm(data,
+                                 gamma,
+                                 beta,
+                                 moving_mean,
+                                 moving_var,
+                                 axis=1,
+                                 epsilon=epsilon,
+                                 center=center,
+                                 scale=scale)[0]
+    return _impl
+
+def _transpose():
+    def _impl(inputs):
+        data = inputs[0]
+
+        if isinstance(data, _expr.Var):
+            ndims = len(get_tensor_from_relay_var(data))
+        elif isinstance(data, (_expr.Call, _expr.TupleGetItem)):
+            ndims = _infer_type(data).checked_type.shape
+        else:
+            ndims = data.shape
+
+        if isinstance(data, tvm.ndarray.NDArray):
+            ndims = len(data.shape)
+        axes = list(range(ndims))
+
+        num_inputs = len(inputs)
+
+        if num_inputs == 1:
+            if ndims >= 2:
+                axes[-1] = ndims - 2
+                axes[-2] = ndims - 1
+            if not isinstance(data, _expr.Var):
+                data = _expr.const(data, dtype='float32')
+
+        elif num_inputs == 3:
+            parse = lambda i: ndims * (i < 0) + i
+            src, dst = [parse(int(inputs[i])) for i in [1, 2]]
+            axes[src] = dst
+            axes[dst] = src
+        else:
+            axes = inputs[1]
+        return _op.transform.transpose(data, axes)
+    return _impl
+
+def _flatten():
+    def _impl(inputs):
+        data = inputs[0]
+        return _op.nn.batch_flatten(data)
+    return _impl
+
+def _dense():
+    def _impl(inputs):
+        use_bias = False
+
+        if isinstance(inputs[0], _expr.Var):
+            use_bias = True
+
+        data = inputs[1]
+        weight = inputs[2]
+        beta = int(inputs[3])
+        alpha = int(inputs[4])
+
+        if isinstance(alpha, int) and isinstance(data, (_expr.Call, _expr.TupleGetItem)):
+            alpha = _expr.const(alpha, dtype='float32')
+            data *= alpha
+
+        if isinstance(beta, int) and isinstance(weight, (_expr.Call, _expr.TupleGetItem)):
+            beta = _expr.const(beta, dtype='float32')
+            weight *= beta
+
+        weight_out = _op.transform.transpose(weight, axes=[1, 0])
+
+        units = _infer_type(weight_out).checked_type.shape[0]
+        dense_out = _op.nn.dense(data, weight_out, units=units)
+
+        if use_bias:
+            bias = inputs[0]
+            return _op.nn.bias_add(dense_out, bias)
+        else:
+            return dense_out
+    return _impl
+
+def _size():
+    def _impl(inputs):
+        axis = int(inputs[1])
+        if isinstance(inputs[0], _expr.Var):
+            shape = get_tensor_from_relay_var(inputs[0])
+        else:
+            shape = _infer_type(inputs[0]).checked_type.shape
+        return shape[axis]
+    return _impl
+
+def _numtotensor():
```

Comment:
In this case, can we simply create a `relay.const`?

Reply:
Yeah, I think for the current models we test against, this doesn't do much but it may be helpful in future iterations.